backend/dal/Helpers/PimsMigrationSqlGenerator.cs

```diff
@@ -19,10 +19,6 @@ public class PimsMigrationSqlGenerator : SqlServerMigrationsSqlGenerator
     {
```

Comment:
Note that this file was the cause of the previous strange behaviour from EF with defaults.

there are two issues with the original implementation that I've noted below.